tools/update_required_branch_checks.py

```diff
@@ -69,7 +69,7 @@ def initialize_repo_connection():
 
 def get_checks_for_branch(repo, branch: str) -> None:
   prs = repo.get_pulls(
-      state='closed', sort='updated', base=branch, direction='desc')
+      state='closed', sort='merged', base=branch, direction='desc')
```

Comment:
If we do do this, let's put a comment saying that this isn't officially listed as a value and that it may break to hopefully give some indication to someone looking in the future

Reply:
Added a comment explaining the use of the undocumented value and what the alternative would be. Could be useful if they drop support for it.